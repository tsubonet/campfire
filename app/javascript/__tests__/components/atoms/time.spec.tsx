import * as React from 'react'
import Time from '../../../components/atoms/time'
import * as renderer from 'react-test-renderer'

test('render test', () => {
  const component = renderer.create(<Time>2018-05-18T11:52:29.241Z</Time>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
