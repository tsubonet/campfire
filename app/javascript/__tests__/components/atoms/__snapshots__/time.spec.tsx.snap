// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<time
  dateTime="2018-05-18T20:52"
>
  05月18日(金)20:52
</time>
`;
